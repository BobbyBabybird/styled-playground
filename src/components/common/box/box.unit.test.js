import React from 'react'
import renderer from 'react-test-renderer'
import Box from './index'
import 'jest-styled-components'

test('<Box /> background color is correct', () => {
  const tree = renderer.create(<Box />).toJSON()
  expect(tree).toMatchSnapshot()
  expect(tree).toHaveStyleRule('background-color', 'transparent')
})
