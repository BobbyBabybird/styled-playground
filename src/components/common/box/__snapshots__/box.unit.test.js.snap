// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Box /> background color is correct 1`] = `
.c0 {
  background-color: transparent;
}

<div
  className="c0"
/>
`;
